pkg/statdb/statdb.go

```diff
@@ -35,8 +36,8 @@ func NewServer(driver, source string, logger *zap.Logger) (*Server, error) {
 		return nil, err
 	}
 
-	_, err = db.Exec(db.Schema())
-	if err != nil && !strings.Contains(err.Error(), "already exists") {
+	err = migrate.Create("statdb", db)
```

Comment:
We still want `statdb.NewServer()` to not return an error if the database already exists. In that case, we just want to use the already existing db.

Reply:
Feel free to dismiss this, as it's something that could be implemented in a separate PR

Reply:
The problem is that there isn't a good way to detect whether it's an old database or table created by some other schema.

Let's say one schema does `CREATE TABLE something (id int)` and some schema adds `CREATE TABLE something (name text)`. It would still be nice if it failed, but we won't have an existing entry for one of them -- effectively we get a, "table already exists" error, but it should be a conflict.

Of course, it doesn't warn when you are trying to recreate the same sql table (assuming the schema table has been populated).

Reply:
Ok, that makes sense. Maybe the db creation should just move to a separate function within statdb. I'll add a ticket for that.

Reply:
It is fine, it won't error when it is called multiple times.